github/orgs_members.go

```diff
@@ -59,7 +59,7 @@ type ListMembersOptions struct {
 	// Possible values are:
 	//     all - all members of the organization, regardless of role
 	//     admin - organization owners
-	//     member - non-organization members
+	//     member - non-owner members
```

Comment:
The documentation of role values at https://developer.github.com/v3/orgs/members/#members-list says:

* `member` - Non-owner organization members.

Would using that phrasing be better?

Reply:
Perfect